src/api.js

```diff
@@ -2379,12 +2379,9 @@ api.declare({
     provisionerId,
     workerType,
     expires: Entity.op.greaterThan(new Date()),
+    quarantineUntil: Entity.op.greaterThan(new Date(quarantineUntil || -8640000000000000)),
```

Comment:
probably better to not include this condition when it's not specified - it might allow Azure to return results more efficiently.